submissions/gorodiskiy/HTML-CSS-Popup/css/main.css

```diff
@@ -0,0 +1,187 @@
+* {
+    box-sizing: border-box;
+    -webkit-appearance: none;
```

Comment:
This is not a good rule for style reset.
This rule turned off the default behavior of all elements, better applied it pointwise for the required elements.